docs/next.config.js

```diff
@@ -32,39 +22,36 @@ logInfo(`Copied latest Expo SDK version from v${version}`);
 module.exports = {
   trailingSlash: true,
   pageExtensions: ['js', 'jsx', 'ts', 'tsx', 'md', 'mdx'],
+  compiler: { emotion: true },
+  swcMinify: true,
+  typescript: {
+    ignoreBuildErrors: true,
+  },
```

Comment:
This addition makes me a little bit worried, is this really needed? Ref: https://nextjs.org/docs/api-reference/next.config.js/ignoring-typescript-errors